glib/src/object.rs

```diff
@@ -913,66 +967,52 @@ macro_rules! glib_object_wrapper {
             }
         }
 
-        impl $crate::types::StaticType for $name {
+        impl $(<$($generic $(: $bound $(+ $bound2)*)?),+>)? $crate::types::StaticType for $name $(<$($generic),+>)? {
             fn static_type() -> $crate::types::Type {
                 #[allow(unused_unsafe)]
                 unsafe { $crate::translate::from_glib($get_type_expr) }
             }
         }
 
-        impl<T: $crate::object::ObjectType> std::cmp::PartialEq<T> for $name {
```

Comment:
Why did you remove these?

Reply:
I will double check this, could be a mistake when resolving conflicts in a rebase

Reply:
As far as I can see I just moved them up a few lines to group them together with the other manually implemented traits